signalfx-forwarder-app/app.manifest

```diff
@@ -5,7 +5,7 @@
     "id": {
       "group": null, 
       "name": "signalfx-forwarder-app", 
-      "version": "2.0.0"
+      "version": "2.0.1"
```

Comment:
Not sure this is needed since we are still validating the 2.0.0 version.

Reply:
I thought there was an issue when we tried to overwrite an existing version in splunkbase, but maybe I'm remembering incorrectly. I'll revert to 2.0.0 and try when we're ready.